UselessMine.cpp

```diff
@@ -489,6 +539,17 @@ void UselessMine::reloadDeathMessages()
     }
 }
 
+//reload the defusal messages
+void UselessMine::reloadDefusalMessages()
```

Comment:
This function isn't declared in the class definition. See line ~178 where `reloadDeathMessages()` is declared